frontend/src/components/settings/ManageUserDialog.vue

```diff
@@ -151,7 +150,10 @@ export default {
       rules,
       isDisplayed: false,
       isLoading: false,
-      facilities: [],
+      user: {},
+      selectedFacilityIds: [],
+      facilitiesToAdminister: [],
+      facilitiesUser: [],
       userOperationType: '', // Can be 'adding', 'updating'
```

Comment:
If there's no precedent can we just use 'add'/'update'?

Reply:
Updated.